### PR TITLE
merge FVM hackathon cheat sheet in here.

### PR DESCRIPTION
**Pull request by longfeiWan9**
- Update some FVM-related info in `README.md`
- Create a page that includes all the information from the FVM hackathon cheat sheet.
- Re-organize the info and link `fvm.md` in `README.md`

### CODE REVIEW

**Comment by longfeiWan9** [comment]
@LesnyRumcajs I guess you have the write access to this repo. Can you please review this PR and merge it if it is good? 

@trruckerfling  please review it as well.

File: README.md [hunk 1]
```diff
@@ -41,54 +43,60 @@ A curated list of useful resources for Filecoin
...
-- [Filecoin Foundation YouTube channel](https://www.youtube.com/@filecoinfoundation) - Talks from Filecoin conferences/events
-- [Filecoin Project YouTube channel](https://www.youtube.com/@filecoinfoundation) - Talks/recordings from the Filecoin ecosystem
-- [How Filecoin Actors Work](https://www.youtube.com/watch?v=9JbwbTPonv0) - Deep dive into the actors by [zenground0](https://github.com/ZenGround0)
-- [The EVM-compatibility of FVM aka FEVM](https://www.youtube.com/watch?v=lgUMVhM3FIM)
+### 📄 Introduction
 
-### 🦝 Other resources
-- [Launchpad Curriculum](https://curriculum.pl-launchpad.io/) - PL Launchpad bootcamp resources
+- [FVM Docs](https://docs.filecoin.io/fvm) - where to get start learning about FVM.
+- [What is FEVM](https://docs.filecoin.io/smart-contracts/fundamentals/filecoin-evm-runtime/) - welcome EVM hacker, here's what allows you to build on Filecoin.
```
Comment (LesnyRumcajs) [inline README.md]: The description is a bit vague - can we summarize it a bit?

File: README.md [hunk 2]
```diff
@@ -41,54 +43,60 @@ A curated list of useful resources for Filecoin
...
 ### 👛 Wallets
 - [Brave Wallet](https://brave.com/wallet/)
 - [GLIF Wallet](https://wallet.glif.io/)
 
-## Resources
+## FVM(Filecoin Virtual Machine)
 
-### 📰 Articles
-- [How storage and retrieval deals work on Filecoin](https://filecoin.io/blog/posts/how-storage-and-retrieval-deals-work-on-filecoin/)
+📢 **[FVM cheatsheet](./fvm.md)** - everything you need to know about FVM.
```
Comment (LesnyRumcajs) [inline README.md]: `everything` is a bit vague, it'd be great to make it a more formal description.

File: fvm.md [hunk 3]
```diff
@@ -0,0 +1,173 @@
+<img src="https://bafybeiesyiylnbxe5lrim35mwcwckoipn6eg32kpn6nibfdgb4vkmvzsta.ipfs.w3s.link/FVM%20logo.png" width="500" >
```
Comment (LesnyRumcajs) [inline fvm.md:1]: Let's add a mention that it was copied from https://github.com/filecoin-project/community/discussions/585 and give correct attribution.

**Review by LesnyRumcajs — COMMENTED** [review]
Thanks!

**Comment by longfeiWan9** [comment]
@LesnyRumcajs thanks for your comments. updated them all.

**Review by LesnyRumcajs — APPROVED** [review]

**Pull request merged**